types/src/types.ts

```diff
@@ -577,6 +577,104 @@ export interface PasswordOAuthFlow {
 // --8<-- [end:PasswordOAuthFlow]
 
 // --8<-- [start:JSONRPCMessage]
```

Comment:
Can you put your change above this line? 

```
// --8<-- [start:JSONRPCMessage]
```
It's used for Code Snippets in the documentation.